### PR TITLE
23967 Make spinner pre-empt action button for in-process filings

### PR DESCRIPTION
**Pull request by leodube-aot**
*Issue #:* /bcgov/entity#23967 and /bcgov/entity#23636

*Description of changes:*
- Fixed bug where spinner would show along with action button for in process filings.

_Before_
Screenshot 2024-11-05 at 3 55 21 PM

_After_
Screenshot 2024-11-05 at 4 08 51 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-699-4grej6ci.web.app

**Comment by JazzarKarim** [comment]
Don't forget to update package version and do an `npm i` Leo please.

File: src/components/Dashboard/TodoList.vue [hunk 1]
```diff
@@ -352,7 +352,7 @@
                 </template>
 
                 <!-- default task -->
-                <template v-if="item.isDefaultTask">
+                <template v-else-if="item.isDefaultTask">
```
Comment (severinbeauvais) [inline src/components/Dashboard/TodoList.vue:355]: Looks right. This block (and all blocks below) need to be mutually exclusive, per line 343.

👍

**Review by severinbeauvais — APPROVED** [review]

**Review by JazzarKarim — APPROVED** [review]
LGTM 👍

**Review by eason-pan-bc — APPROVED** [review]
LGTM 😃

**Pull request merged**